.travis.yml

```diff
@@ -72,8 +72,7 @@ jobs:
         provider: script
         script: ssh omniledger@c4dtsrv1.epfl.ch bin/update.sh
       on:
-        branch:
-          main
+        branch: main
```

Comment:
I'm surprised, because those two should be equivalent.

Perhaps:
```
    deploy:
      ...
      on:
        branch: main
```
i.e. `on:` **indented** under `deploy:`?

Reply:
Thanks! I think that's it. I _did_ think that it's strange it didn't work that way. But indenting `on` under `deploy` looks definitely better!

https://docs.travis-ci.com/user/deployment/#conditional-releases-with-on

I'm not sure if I should blame yaml for this, or be happy that yaml is so concise...